repo-projects/runtime.proj

```diff
@@ -15,6 +15,9 @@
     <SkipSetTargetOS>true</SkipSetTargetOS>
     <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)os $(TargetOS)</BuildArgs>
 
+    <!-- Don't publish Windows PDBs from the runtime repo. They're not used and they slow down publishing. -->
+    <PublishWindowsPdb>false</PublishWindowsPdb>
```

Comment:
Can't we have this directly in the runtime repo, i.e. in its Publishing.props file?

Reply:
I was just following consistency with how we currently do it for sdk and deployment tools. I'll move those into the repos as well.

Reply:
Yes I think that would be better to reduce differences and keep the VMR specialness as small as possible.